efficientdet/efficientdet_arch.py

```diff
@@ -401,6 +409,7 @@ def build_feature_network(features, config):
             conv_after_downsample=config.conv_after_downsample,
             use_native_resize_op=config.use_native_resize_op,
             conv_bn_relu_pattern=config.conv_bn_relu_pattern,
+            use_tpu=config.use_tpu,
```

Comment:
Should you update the hparams_config.py to add use_tpu?

Reply:
No, config get ```use_tpu``` from params which get ```use_tpu``` from FLAGS.

Reply:
If use_tpu is not in config, other scripts like model_inspect.py or inference.py may fail (also inconvenient if people directly use efficientdet_arch rather than main).

An alternative:  change it to  "use_tpu=config.get('use_tpu', None)". WDYT?

Reply:
My fault, I add it to hparams